README.md

```diff
@@ -164,5 +206,8 @@ Write a route in sinatra that will print "hello world" in the web browser at the
 
 Your answer:
 ```
-Replace this with your answer
+get '/oh hello'
+ print "hello world"
+end
+
 ```
```

Comment:
in sinatra we know longer want to print or puts. Also path names must be all one word, so something like this:

```ruby
get '/oh_hello' do
  "hello world"
end
```